maptk/local_geo_cs.cxx

```diff
@@ -65,26 +66,33 @@ namespace maptk {
 
 /// Constructor
 local_geo_cs
-::local_geo_cs(algo::geo_map_sptr alg)
-: geo_map_algo_(alg),
-  utm_origin_(0.0, 0.0, 0.0),
-  utm_origin_zone_(-1)
+::local_geo_cs()
+: geo_origin_(),
+  origin_alt_(0.0)
 {
 }
 
 
+/// Set the geographic coordinate origin
+void
+local_geo_cs
+::set_origin(const geo_point& origin)
+{
+  // convert the origin point into WGS84 UTM for the appropriate zone
+  vector_2d lon_lat = origin.location( SRID::lat_lon_WGS84 );
```

Comment:
FYI, if no conversion plugin is loaded, this will throw... is that okay, or do there need to be some exception checks? (Maybe not; I see later there is a check that a conversion functor is registered, but I still wonder because there were checks in this vicinity that the old geo_map algorithm was non-null.)